lib/stat_tracker.rb

```diff
@@ -57,13 +57,39 @@ def percentage_visitor_wins
     (away_win?.to_f / @games.size).round(2)
   end
 
+  def percentage_ties
+    tie_games = @game_teams.find_all do |game|
+      game.game_result.include?("TIE")
+    end
+    results = tie_games.length.to_f / @game_teams.length.to_f
+    results.round(2)
+  end
+
+  def count_of_games_by_season
+    ids = []
+    only_ids = []
+      @games.find_all do |game|
+        ids = game.season_id
+        only_ids << ids
+      end
+    uniq_ids = only_ids.uniq
+    count_games = []
+    final_count = Hash.new
+      uniq_ids.each do |id|
+        count_games = @games.find_all {|game|game.season_id.include?(id)}
+        final_count[id] = count_games.length
```

Comment:
I believe you can also use `count` here instead of `length` as you do below. This still works correctly though.